### PR TITLE
Add script to build AMI via SSM

### PR DESCRIPTION
**Pull request by michaelsauter**
Useful for environments behind proxy that cannot establish connections on port 22.

### CODE REVIEW

File: ods-devenv/packer/build-ods-ami-ssm.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -eu
+set -o pipefail
+# build-ods-ami-ssm.sh builds an ODS image using Packer over SSM.
+# It depends on a prebuilt CentOS7 AMI which has SSM installed.
```
Comment (stitakis) [inline ods-devenv/packer/build-ods-ami-ssm.sh:5]: This line mentions that this scripts depends on a prebuild CentOS7 with SSM installed, however in the code I can't see why SSM is required. Is there a hidden dependency? If yes, can you please a line explaining this?

Comment (michaelsauter) [reply ods-devenv/packer/build-ods-ami-ssm.sh:5]: It's in the packer JSON. That is configured to use SSM instead of SSH.

**Review by stitakis — APPROVED** [review]

File: ods-devenv/packer/build-ods-ami-ssm.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# To use this in your AWS account, you must have an IAM instance profile setup
+# as described in https://docs.aws.amazon.com/systems-manager/latest/userguide/setup-instance-profile.html#instance-profile-add-permissions.
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+iam_instance_profile="SSMInstanceProfile"
```
Comment (georgfedermann) [inline ods-devenv/packer/build-ods-ami-ssm.sh:11]: awesome! works like a charme.
Would be great if iam_instance_profile could be configured via arguments, too.

File: ods-devenv/packer/build-ods-ami-ssm.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+profile="default"
+region="eu-west-1"
+instance_type="m5ad.4xlarge"
+ods_branch="master"
+
```
Comment (georgfedermann) [inline ods-devenv/packer/build-ods-ami-ssm.sh:16]: also to have the (private) VPC subnet configurable where the build box should be run.

**Comment by stitakis** [comment]
@michaelsauter do we need this change. Otherwise let us close it.

**Comment by michaelsauter** [comment]
@stitakis I don't know to be honest. I never use the box (except for this one time). It seems like no-one else needed it so let's close it.

**Pull request closed without merging**